build-system/tasks/check-types.js

```diff
@@ -268,7 +302,9 @@ async function checkTypes() {
   // Use the list of targets if provided, otherwise check all targets
   const targets = argv.targets
     ? argv.targets.split(/,/)
-    : Object.keys(TYPE_CHECK_TARGETS);
+    : Object.keys(TSC_TYPECHECK_TARGETS).concat(
+        Object.keys(CLOSURE_TYPE_CHECK_TARGETS)
+      );
```

Comment:
Identical but maybe slightly easier to read? Not certain:
```suggestion
    : Object.keys({
        ...TSC_TYPECHECK_TARGETS,
        ...CLOSURE_TYPE_CHECK_TARGETS,
      });
```
```suggestion
    : Object.keys(TSC_TYPECHECK_TARGETS).concat(
        Object.keys(CLOSURE_TYPE_CHECK_TARGETS)
      );
```

Reply:
I actually wrote it both ways at different times. It more terse...but makes it seem like values matter